### ISSUE TITLE
Button does not have discernible text

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#1001]
### Error in spec/features/accessibility_spec.rb

accessibility when visiting main page complies with WCAG

       button-name: Buttons must have discernible text (critical)
       https://dequeuniversity.com/rules/axe/4.3/button-name?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: #search
               HTML: <button type="submit" class="btn btn-primary" id="search"><span class="submit-search-text d-none">Search</span><i class="fa fa-search" aria-hidden="true"></i></button>
               Fix any of the following:
               - Element does not have inner text that is visible to screen readers
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has no title attribute
               - Element's default semantics were not overridden with role="none" or role="presentation"
